general/templates/general/base.html

```diff
@@ -35,7 +35,7 @@
 
         <header class="brandbar container">
             <div class="navbar yoda-topnavbar">
-                <a class="navbar-brand" href="{{ config.get('LOGO_URL') }}">
```

Comment:
Logo url is intended to be https://www.uu.nl, as is the behaviour for other UU web applications.
After investigating this is not the case for the php portal of Yoda and could be deleted as such, but this needs verification as the reasoning for deleting teh LOGO_URL variable is not the case.